public/index.js

```diff
@@ -99,37 +99,6 @@ ReactDOM.render(
       </cmVariableContext.Provider>
     </pre>
 
-    <h2>Custom Theme: tomorrow-night</h2>
```

Comment:
Removed this from the demo page because `tomorrow-night` isn't being loaded within the library (because it's a custom theme) so the styles on these were always jank.